.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,23 +6,29 @@ about: Create a report to help us improve Zephyr
 
 **Describe the bug**
 A clear and concise description of what the bug is.
+What have you tried to diagnose or workaround this issue?
```

Comment:
The bug report and feature request changes should be in the previous commit

Reply:
remove PR template for now, causing issues when submitting from the command line.